sequencer/src/api/options.rs

```diff
@@ -151,7 +161,9 @@ impl Options {
             let status_api = status::define_api(&Default::default(), bind_version)?;
             app.register_module("status", status_api)?;
 
+            // Initialise only here because Extensivle Data Source doesn't impl the EventsSource trait.
```

Comment:
This means we silently ignore the hotshot events module when other modules are present. Is this what the PR in hotshot-query-service is intended to fix?